ofl/notosansmono/METADATA.pb

```diff
@@ -1,7 +1,7 @@
 name: "Noto Sans Mono"
 designer: "Google"
 license: "OFL"
-category: "MONOSPACE"
+category: "SANS_SERIF"
```

Comment:
Category should be Monospace imo, but sans_serif can be added as a secondary one (so in first place)